<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addinfo</class>
 <widget class="QDialog" name="addinfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>460</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="StudentInfo">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>30</y>
     <width>281</width>
     <height>331</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="9" column="1">
     <widget class="QLineEdit" name="AwardEdit_2"/>
    </item>
    <item row="7" column="1">
     <widget class="QSpinBox" name="ClassEdit_2">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>20</number>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_20">
      <property name="text">
       <string>*学院</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="label_22">
      <property name="text">
       <string>班级</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QSpinBox" name="AgeEdit_2">
      <property name="minimum">
       <number>14</number>
      </property>
      <property name="maximum">
       <number>40</number>
      </property>
     </widget>
    </item>
    <item row="11" column="1">
     <widget class="QLineEdit" name="NativePlaceEdit_2"/>
    </item>
    <item row="8" column="1">
     <widget class="QLineEdit" name="PhoneNumberEdit_2"/>
    </item>
    <item row="10" column="0">
     <widget class="QLabel" name="label_28">
      <property name="text">
       <string>处分</string>
      </property>
     </widget>
    </item>
    <item row="10" column="1">
     <widget class="QLineEdit" name="PunishEdit_2"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_19">
      <property name="text">
       <string>年龄</string>
      </property>
     </widget>
    </item>
    <item row="11" column="0">
     <widget class="QLabel" name="label_29">
      <property name="text">
       <string>籍贯</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QComboBox" name="CollegeEdit_2">
      <item>
       <property name="text">
        <string>软件学院</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>职业教育学院</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>城市文化学院</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>国际商学院</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_21">
      <property name="text">
       <string>*专业</string>
      </property>
     </widget>
    </item>
    <item row="9" column="0">
     <widget class="QLabel" name="label_27">
      <property name="text">
       <string>奖励</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QComboBox" name="GenderEdit_2">
      <item>
       <property name="text">
        <string>男</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>女</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLineEdit" name="MajorEdit_2"/>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="label_23">
      <property name="text">
       <string>*手机号</string>
      </property>
     </widget>
    </item>
    <item row="8" column="2">
     <widget class="QLabel" name="PhoneTip">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>11位</string>
      </property>
     </widget>
    </item>
    <item row="12" column="1">
     <widget class="QLineEdit" name="ParentsInfoEdit_2"/>
    </item>
    <item row="12" column="0">
     <widget class="QLabel" name="label_30">
      <property name="text">
       <string>家长信息</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_16">
      <property name="text">
       <string>*学号</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="StuNoTip">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>6位</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="NameEdit_2"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_18">
      <property name="text">
       <string>性别</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="StuNoEdit_2">
      <property name="placeholderText">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_17">
      <property name="text">
       <string>*姓名</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLabel" name="label">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>30字符以内</string>
      </property>
     </widget>
    </item>
    <item row="6" column="2">
     <widget class="QLabel" name="label_2">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>50字符以内</string>
      </property>
     </widget>
    </item>
    <item row="9" column="2">
     <widget class="QLabel" name="label_3">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>300字符以内</string>
      </property>
     </widget>
    </item>
    <item row="10" column="2">
     <widget class="QLabel" name="label_4">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>300字符以内</string>
      </property>
     </widget>
    </item>
    <item row="11" column="2">
     <widget class="QLabel" name="label_5">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>30字符以内</string>
      </property>
     </widget>
    </item>
    <item row="12" column="2">
     <widget class="QLabel" name="label_6">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>100字符以内</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>380</y>
     <width>195</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="add">
      <property name="text">
       <string>添加</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cancel">
      <property name="text">
       <string>取消</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
